packages/aila/src/utils/moderation/moderationErrorHandling.ts

```diff
@@ -24,7 +24,7 @@ export async function handleThreatDetectionError(
   SafetyViolations = defaultSafetyViolations,
 ): Promise<ErrorDocument | ActionDocument> {
   await safelyReportAnalyticsEvent({
-    eventName: "helicone_threat_detected",
```

Comment:
Do we need to make any changes in AVO to change this name?

Reply:
I don't think so because we are sending events directly via Posthog, and not using the Avo client. If we were using the Avo client we would get a typescript error at this point. But worth checking what we want to do here.

Reply:
👍 understood